src/coreclr/gc/gc.cpp

```diff
@@ -25854,7 +25987,7 @@ bool gc_heap::change_heap_count (int new_n_heaps)
             int max_n_heaps = max (old_n_heaps, new_n_heaps);
             budget_per_heap[gen_idx] = Align (total_budget/max_n_heaps, get_alignment_constant (gen_idx <= max_generation));
 
-            dprintf (6666, ("g%d: total budget: %zd budget per heap: %zd", gen_idx, total_budget, budget_per_heap[gen_idx]));
+            dprintf (3, ("g%d: total budget: %zd budget per heap: %zd", gen_idx, total_budget, budget_per_heap[gen_idx]));
```

Comment:
keep this and the next change from 6666 to 3?

Reply:
yeah, I would keep this change because it can be so verbose